### PR TITLE
Tidy ups on Ownable stuff

### LINKED ISSUE #339: Cleanup core-contracts/ownable

### PR DESCRIPTION
**Pull request by ajsantander**
Fix #339

Deleted OwnerModule tests in core-modules because they are not really needed. They don't increase coverage, and the mock is no longer needed.

### CODE REVIEW

File: packages/core-contracts/contracts/ownership/Ownable.sol [hunk 1]
```diff
@@ -14,36 +14,36 @@ contract Ownable is OwnableMixin, CommonErrors {
     event OwnerChanged(address oldOwner, address newOwner);
```
Comment (ajsantander) [inline packages/core-contracts/contracts/ownership/Ownable.sol:14]: Simply removed unnecessary wrapper functions on this one.

File: packages/core-modules/contracts/modules/OwnerModule.sol [hunk 2]
```diff
@@ -3,4 +3,7 @@ pragma solidity ^0.8.0;
```
Comment (ajsantander) [inline packages/core-modules/contracts/modules/OwnerModule.sol:3]: Just a linter fix.

**Review by mjlescano — APPROVED** [review]

**Review by i-stam — APPROVED** [review]
looks good, the owner module stuff should have been deleted in the previous PR.
Not a big of having the whole `_ownableStorage().nominatedOwner` everywhere but it's ok since it removed the internal getter functions

**Review by leomassazza — APPROVED** [review]
LGTM

**Pull request merged**